build/release/README.md

```diff
@@ -8,8 +8,6 @@ The workflow at a high level for minor releases is as follows:
 2. Create the new release branch with a beta release when the commits are winding down
 3. When all the release criteria is met we tag the final release
 
-GitHub actions provide all the jobs to complete the release -- there is no need to perform any of the release tasks on a build/dev machine.
```

Comment:
This was the goal, but unfortunately couldn't get the github actions to tag and trigger a release as expected.